e2e-tests/pipeline_validation.sh

```diff
@@ -220,6 +220,45 @@ function test_parquet_sink() {
   fi
 }
 
+
+#################################################
+# Function that counts resources in materialized view parquet files and
+# compares output to what is in source FHIR server
+# Globals:
+#   HOME_PATH
+#   PARQUET_SUBDIR
+#   TOTAL_TEST_PATIENTS
+#   TOTAL_TEST_ENCOUNTERS
+#   TOTAL_TEST_OBS
+#   OPENMRS
+#################################################
+function test_parquet_view_sink() {
```

Comment:
We might have to invoke this to execute, like we do for other functions at the end of this file. This method looks like a duplicate code of the function `test_parquet_sink`, see if it can be refactored to a common function.

Reply:
On the count part, we might have to see if the records are mapped 1:1 from the source to the flat tables and accordingly set the validation. My assumption is that there will be more records in the views.